docs/sql-server/fabric-database/fabric-mirrored-databases.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Microsoft Fabric mirrored databases"
-description: Learn about the mirrored databases in Microsoft Fabric, from SQL Server and Azure SQL Database, to allow for real-time analytics of data in Microsoft Fabric.
+title: "Mirroring and Database Mirroring in Microsoft Fabric"
```

Comment:
Mirroring and Database Mirroring are not separate concepts. There's Mirroring. And databases can be mirrored. 

We're also trying to make sure we are not confusing the newer Fabric mirrored database feature with the deprecated Database Mirroring feature in SQL Server, which is unrelated.